### PR TITLE
Add getSort() in OffserPaginator::class.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Paginator/OffsetPaginator.php [hunk 1]
```diff
@@ -188,6 +189,11 @@ public function getPageSize(): int
         return $this->pageSize;
     }
 
+    public function getSort(): ?Sort
```
Comment (samdark) [inline src/Paginator/OffsetPaginator.php:192]: I think it should be added to `PaginatorInterface` as well, right?

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Paginator/OffsetPaginator.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Paginator/OffsetPaginator.php:192]: 👍

**Review by roxblnfk — CHANGES_REQUESTED** [review]

File: src/Paginator/OffsetPaginator.php [hunk 2]
```diff
@@ -188,6 +189,11 @@ public function getPageSize(): int
...
+    {
+        return $this->dataReader->getSort();
```
[the remark this answers is not in the document]
Comment (samdark) [reply src/Paginator/OffsetPaginator.php:194]: Else return null, yes.

File: src/Paginator/OffsetPaginator.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Paginator/OffsetPaginator.php:192]: done.

**Comment by samdark** [comment]
👍

**Pull request merged**